core/src/consensus/stake/mod.rs

```diff
@@ -1174,7 +1212,8 @@ mod tests {
         stake.execute(&action.rlp_bytes(), &mut state, &delegator, &delegator_pubkey).unwrap();
 
         for current_term in 0..=released_at {
-            on_term_close(&mut state, current_term).unwrap();
+            on_term_close(&mut state).unwrap();
```

Comment:
If we make `on_term_close` handle increasing term id, just calling `term_close_until(..., released_at)` will replace this loop.